src/frontend/components/VideoPlayerConnected/VideoPlayerConnected.tsx

```diff
@@ -19,14 +25,41 @@ export const mapStateToProps = (
   { video }: VideoPlayerConnectedProps,
 ) => ({
   jwt: state.context.jwt,
+  timedtexttracks: getTimedTextTracks(state),
   video:
     (state.resources[modelName.VIDEOS]!.byId &&
       state.resources[modelName.VIDEOS]!.byId[(video && video.id) || '']) ||
     video,
 });
 
+/** Create a function that adds a bunch of timedtexttracks in the store. */
```

Comment:
I'd say it triggers a get to the API for the timedtexttracks rather than "adds" them